<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddHostWindow</class>
 <widget class="QMainWindow" name="AddHostWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>135</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>373</width>
    <height>135</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>373</width>
    <height>135</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Add New Entry</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gfx.qrc">
    <normaloff>:/icons/res/icon.png</normaloff>:/icons/res/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="txtHost">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Host Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtIP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Enter a valid IP address (0.0.0.0 - 255.255.255.255)</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="placeholderText">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fill out the following text boxes and click OK to add a new DNS record</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="gfx.qrc"/>
 </resources>
 <connections/>
</ui>
